packages/blockchain-environment-v1/package.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Could we remove all of the unnecessary information from this - will make it easier to read/edit!